plugins/workspace-backpack/src/index.js

```diff
@@ -119,6 +132,25 @@ export class Backpack {
      * @private
      */
     this.initialized_ = false;
+
+    /**
+     * A list of XML (stored as strings) representing blocks in the backpack.
+     * @type {!Array<string>}
+     * @private
+     */
+    this.contents_ = [];
+
+    /**
+     * The maximum items that can be stored on the backpack.
+     */
+    this.maxItems_ = 32;
+
+    /**
+     * The backpack flyout. Initialized during init.
+     * @type {Blockly.IFlyout|undefined}
+     * @package
+     */
+    this.flyout = undefined;
```

Comment:
Why not null here?

Reply:
I set it to `undefined` because I don't want to assign a value. Setting it to undefined communicates that the variable has been declared but not defined.
That way I'm saying that this object is never null after it is initialized.